pipelines/tekton/build-container-image-pipeline/build-container-image-pipeline.yaml

```diff
@@ -19,7 +19,37 @@ spec:
     type: string
   - name: containerfileRelativePath
     type: string
+  - name: modelRelativePath
+    type: string
+  - name: fetch-git-model
+    type: string
+  - name: fetch-s3-model
+    type: string
+  - name: git-model-repo
+    type: string
+  - name: git-revision
+    type: string
   tasks:
+  - name: git-clone-model-repo
+    taskRef:
+      kind: ClusterTask
```

Comment:
`ClusterTask` is apparently deprecated 

https://tekton.dev/docs/pipelines/tasks/#task-vs-clustertask